src/cascadia/TerminalSettingsModel/GlobalAppSettings.idl

```diff
@@ -108,5 +108,6 @@ namespace Microsoft.Terminal.Settings.Model
         void AddTheme(Theme theme);
         INHERITABLE_SETTING(ThemePair, Theme);
         Theme CurrentTheme { get; };
+        Boolean ShouldUsePersistedLayout();

```

Comment:
Meh, that check was being used in a bunch of places, so I thought it too brittle to have the same logic everywhere. Thought I should just put it in once place. 

In like, the next branch, I'm also changing this to `FirstWindowPreference() == FirstWindowPreference::PersistedWindowLayout && !IsolatedMode()`, so it seems like it will make more sense in the next one 🤷

Reply:
does the settings model know about Isolated Mode? It it can be specified via command line, I would venture the answer is "no"... and therefore the check for it would almost certainly not belong in TSM!